index.bs

```diff
@@ -1957,6 +1977,11 @@ To <dfn>obtain a randomized source response pick rate</dfn> given a [=randomized
 1. Let |numPossibleValues| be the [=set/size=] of |possibleValues|.
 1. Return |numPossibleValues| / (|numPossibleValues| - 1 + e<sup>|epsilon|</sup>).
 
+To <dfn>obtain fake probability for a source with attribution scope</dfn> given a positve 32-bit integer |numSourceStates|, a positve 32-bit integer |numEventStates|, a positve 32-bit integer |numEvents|, a positive and a double |epsilon|:
```

Comment:
Why are we renaming `obtain a randomized source response pick rate`?

Reply:
synced offline, renamed this to `obtain pick rate for a source with attribution scope`.